client/App.jsx

```diff
@@ -87,5 +82,5 @@ export default App;
 
 const Body = styled.div`
   color: black;
-  font-family: Helvetica, sans-serif;
+  font-family: Targetica, "Helvetica Neue", Helvetica, Arial, sans-serif
```

Comment:
Were you able to use the Targetica font? Lemme know if/how you did 'cause I'd like to use it too! :)